locations/pipelines.py

```diff
@@ -29,3 +29,13 @@ def process_item(self, item, spider):
         item['extras'] = existing_extras
 
         return item
+
+class ApplyBrandPipeline(object):
+
+    def process_item(self, item, spider):
+        existing_extras = item.get('extras', {})
```

Comment:
I think we should promote `brand` to a documented optional attribute on `DATA_FORMAT.md` and include it in the "item" so that it can be output without having to use the `extras` field. Then this block can check if the brand is on the spider or in the item, preferring the result from the item.

I think we might want to support `brand:wikidata` to let spiders specify the Wikidata QID for the brand.

Reply:
I imagine we'll want to include a category at some point, too. Maybe we should model this as a dict of attributes that apply to all items output by a spider, with attributes on the item having precedence?

Reply:
> I think we should promote `brand` to a documented optional attribute on `DATA_FORMAT.md` and include it in the "item" so that it can be output without having to use the `extras` field. Then this block can check if the brand is on the spider or in the item, preferring the result from the item.
> 

are you suggesting that the spiders that already put this in extras/brand, should be edited to move it up to item as well?

Reply:
> I think we might want to support `brand:wikidata` to let spiders specify the Wikidata QID for the brand.

👀 I can help!  want to open a new issue to collaborate on this?

Reply:
> are you suggesting that the spiders that already put this in extras/brand, should be edited to move it up to item as well?

Yep!

> I can help! want to open a new issue to collaborate on this?

Hi Bryan! Yea, a new ticket would be great. Thank you!